aio/tools/examples/BUILD.bazel

```
@@ -1,17 +1,56 @@
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("@aio_npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
 
 package(default_visibility = ["//visibility:public"])
 
+EXAMPLE_BOILERPLATE_SRCS = [
+    "example-boilerplate.js",
+    "constants.js",
+] + glob(["shared/**"])
+
+EXAMPLE_BOILERPLATE_DEPS = [
```

I think this might be a little ambiguous. More explicit would be: 
```suggestion
EXAMPLE_BOILERPLATE_CREATE_DEPS = [
```

there is a difference between deps for creating the boilerplate vs. the deps of the boilerplate right?

---

Yeah the two scripts are different. The boilerplate one produces build outputs. It's just incidental that they have the same deps.

---

Regarding the naming, I'm just mimicing the name of the script:

`example-boilerplate.js` -> `EXAMPLE_BOILERPLATE_DEPS`
`create-example.js` -> `CREATE_EXAMPLE_DEPS`

If I add "CREATE" there could be some slight confusion with the other script?